packages/base64/src/decode.js

```diff
@@ -18,7 +18,7 @@ import { monodu64, padding } from './common.js';
  * messages.
  * @returns {Uint8Array} decoded bytes
  */
-const jsDecodeBase64 = (string, name = '<unknown>') => {
+export const jsDecodeBase64 = (string, name = '<unknown>') => {
```

Comment:
It might be worth leaving a comment because this justifies the existence of the thunk at `packages/base64/decode.js` that just exports the portion of this module that is public.